packages/modules/src/modules/epics/ContributionsLiveblogEpic.stories.tsx

```diff
@@ -1,64 +1,28 @@
-import React, { ReactElement } from 'react';
+import React from 'react';
 import { ContributionsLiveblogEpic } from './ContributionsLiveblogEpic';
-import { withKnobs, text, object } from '@storybook/addon-knobs';
-import { StorybookWrapper } from '../../utils/StorybookWrapper';
-import testData from './ContributionsLiveblogEpic.testData';
-import { EpicVariant, Tracking } from '@sdc/shared/types';
+import { Story, Meta } from '@storybook/react';
+import { EpicProps } from '@sdc/shared/types';
+import { props } from './utils/storybook';
+import { WithReminder } from '../banners/contributions/ContributionsBanner.stories';
 
 export default {
     component: ContributionsLiveblogEpic,
-    title: 'Components/ContributionsLiveblogEpic',
```

Comment:
I've refactored this file to bring it in line with other storybook entries, and added a new story for when the cta does not link to the support site